app/controllers/bestuurseenheid/zitting/uittreksels/index.js

```diff
@@ -1,7 +1,12 @@
+/* eslint-disable ember/no-computed-properties-in-native-classes */
```

Comment:
I disabled this rule for now since I don't really know the alternative yet. The computeds know how to deal with the async relationships, but the getter version doesn't so it would require a promise + helper in the template.